### ISSUE TITLE
Unable to load Spring models

### ISSUE THREAD

**Issue by cshreyastech** [WPI-AIM/ambf_addon#6]
Steps:
Blender version:2.83
1) Load psm-spring.yaml model with this plugin

Unable to load model.

addon_utils.disable: ambf_addon not loaded
Modules Installed (ambf_addon) from '/localcodebase/ambf_addon_adnan/ambf_addon/ambf_addon.py' into '/home/shreyas/.config/blender/2.83/scripts/addons'
module changed on disk: '/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py' reloading...
addon_utils.disable: ambf_addon not loaded
addon_utils.disable: ambf_addon not loaded
Modules Installed (ambf_addon) from '/localcodebase/ambf_addon_adnan/ambf_addon/ambf_addon.py' into '/home/shreyas/.config/blender/2.83/scripts/addons'
module changed on disk: '/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py' reloading...
Warning: class AMBF_PG_CollisionShapePropGroup contains a property which should be an annotation!
/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py:4505
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.ambf_rigid_body_collision_shape_radius
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.ambf_rigid_body_collision_shape_height
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.ambf_rigid_body_collision_shape_xyz_dims
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.disable_update_cbs
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.ambf_rigid_body_collision_shape_pointer
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.ambf_rigid_body_collision_shape
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.ambf_rigid_body_collision_shape_axis
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.ambf_rigid_body_linear_shape_offset
    assign as a type annotation: AMBF_PG_CollisionShapePropGroup.ambf_rigid_body_angular_shape_offset
/localcodebase/ambfnags92/ambf/ambf_models/descriptions/multi-bodies/robots/blender-mtm-springy.yaml
Import finished in 0.0007 sec.
Traceback (most recent call last):
  File "/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py", line 3168, in execute
    self.load_body(body_name)
  File "/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py", line 2628, in load_body
    self.load_material(body_data, obj_handle)
  File "/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py", line 2390, in load_material
    mat.specular_intensity = body_data['color components']['specular']['r'] / mat.diffuse_color[0]
ZeroDivisionError: float division by zero

location: <unknown location>:-1

location: <unknown location>:-1
Error: Traceback (most recent call last):
  File "/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py", line 3168, in execute
    self.load_body(body_name)
  File "/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py", line 2628, in load_body
    self.load_material(body_data, obj_handle)
  File "/home/shreyas/.config/blender/2.83/scripts/addons/ambf_addon.py", line 2390, in load_material
    mat.specular_intensity = body_data['color components']['specular']['r'] / mat.diffuse_color[0]
ZeroDivisionError: float division by zero

location: <unknown location>:-1

**Comment by adnanmunawar** [comment]
Hey Shreyas. Thanks for creating this issue. This commit should fix that. 
On a general note, the *-springy.yaml ADF files were created at one point as a test to validate the ADF's compatibility with springs. All I did was that I copied over the original NON-springy ADF files and replaced the joint types with torsional or linear springs. The springy ADF files shared the same meshes as the original files. Over time we have significantly cleaned up these original files and removed many unnecessary meshes. Thus now the springy ADF files are out of date and don't add anything meaningful. We should thus remove these files from the ambf repo.
